src/main.rs

```diff
@@ -31,8 +33,38 @@ use vmm::vmm_config::instance_info::{InstanceInfo, InstanceState};
 
 const DEFAULT_API_SOCK_PATH: &str = "/tmp/firecracker.socket";
 const DEFAULT_INSTANCE_ID: &str = "anonymous-instance";
+const KERNEL_VERSION: f64 = 4.14;
+
+fn is_kvm_writable() -> Result<bool, Error> {
+    let kvm_rw = Command::new("sh")
+        .arg("-c")
+        .arg("[ -r /dev/kvm ] && [ -w /dev/kvm ] && echo OK|| echo FAIL")
+        .output()
+        .expect("failed to execute process");
+    Ok(str::from_utf8(&kvm_rw.stdout[0..2]).unwrap() == "OK")
+}
+fn kernel_version() -> Result<f64, Error> {
+    let kernel_version = Command::new("sh")
+        .arg("-c")
+        .arg("uname -r")
+        .output()
+        .expect("failed to execute process");
+    let s = &kernel_version.stdout[0..4];
+    let _v: f64 = str::from_utf8(&s).unwrap().parse().unwrap();
+    Ok(_v)
+}
+
+fn check_requirements() {
+    if !is_kvm_writable().expect("not able to find /dev/kvm") {
+        panic!("/dev/kvm is not readable nor writable");
```

Comment:
This message is not always accurate since `/dev/kvm` could be readable and not writable or vice-versa.

Reply:
Thanks!, I'll change it to something like this :

"/dev/kvm should be readable and writable by firecracker"